Karambit

CptnCapitalism opened the pull request:
Adds a new melee weapon for the Syndicate, rebalanced based on the feedback I got.

## Why It's Good For The Game
While eswords are robust enough to kill an opponent in a few hits, their sound makes it obvious for anyone nearby that they have a traitor on the ship. This karambit deals less damage than an edagger, but gives agents the ability to "slit" a target's throat,causing heavy bleeding and making them unable to speak for a short time, allowing for a new way of silent murder.

## Changelog
:cl:
add: Added the karambit, a new Syndicate weapon which deals less damage than an edagger, but has a special attack that prevents your target from speaking for a short while.
add: Karambits can be found in Syndicate uplinks under Stealthy and Inconspicuous Weapons, and in the nuke ops lockers
/:cl:

CptnCapitalism committed:
Adds a karambit, a new Syndicate item

A more stealthy, but less powerful alternative to eswords. Still beats most knives found on the station. Can be purchased from the uplink, or found in a nuke ops locker.

CptnCapitalism committed:
Adds a silencing ability for the karambit

Now you can use the karambit to slit an opponent's throat, making them unable to speak for a short time. This won't deal any extra damage.

CptnCapitalism committed:
Rebalanced the karambit

Reduced the base damage to 12 (2/3 of the edagger) for normal attacks, and you have to target the mouth for the "muting" attack, which only deals half the normal damage, but causes heavy bleeding. Also reduced the muting time from 15 to 5 seconds per hit, and it won't stack above 5.

Finally I moved it under kitchen.dm, so now it can be used for butchering, or even in ghetto surgery.

SouDescolado commented:
How broken would this be if you could insta-kill a sleeping target?

CptnCapitalism commented:
> 
> 
> How broken would this be if you could insta-kill a sleeping target?

Not too bad, unless you combine it with a sleepy pen. But that feature could make sense on any sharp object.

optimumtact commented:
Did you make a new PR?

also I still don't think we need more syndicate items

CptnCapitalism commented:
Yes, this is a new PR (sorry, I'm new to using git). At least now I know how to push new commits to an existing one.

CptnCapitalism commented:
Right now if a traitor wants to take out a target silently and doesn't want them to alert the station on comms, he could either buy a sleepypen + a poison kit, or a stechkin + a suppressor. Both of these options cost the traitor half his TCs, but more importantly, neither gives the victim a fighting chance. They are dead in a few seconds, and their body will either be spaced, or hidden in some random locker, effectively ending their round as a human.

This karambit allows the traitor to have a more "loud" loadout, with a chance for taking out single targets "quietly" when needed, while giving the victim a fair shot at fighting their attacker, since the weapon only does 6 damage in the silencing mode.

Nickkoranda commented:
Why not replace the survival knife nukes have with the karambit instead? Lowkey a buff to the caravan ruin since the armory closet spawns in there.

CptnCapitalism commented:
I already added them to the lockers at the nukie outpost where they start, but I could do that too. It deals less damage than the survival knife (12 instead of 15), but if someone wants to go for damage, they'll just use their esword anyway.

Qustinnus commented:
@optimumtact traitor items are easy for new coders to make and can usually be unique/varied enough that they all fill their own niche so I'd say the more the merrier, we can just remove shitty traitor items if it becomes a problem

nemvar commented on code/game/objects/items/kitchen.dm:
```diff
@@ -134,6 +134,37 @@
+
+/obj/item/kitchen/karambit/attack(mob/living/carbon/human/M, mob/living/carbon/human/user)
+	if(user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		force = 6
```
force -= 6

nemvar commented on code/game/objects/items/kitchen.dm:
```diff
@@ -134,6 +134,37 @@
+			if(M.bleed_rate >= 4)
+				M.bleed_rate = 4
+		..()
+		force = 12
```
force += 6

CptnCapitalism commented:
Good idea. I'll change it to "force /= 2" and "force *= 2", since the main idea is to halve the damage for the muting attack.

CptnCapitalism committed:
Whetstone compatibility

Now it can be sharpened with a whetstone to deal up to 16 damage normally, and 8 for a muting attack.

CptnCapitalism committed:
Merge remote-tracking branch 'upstream/master' into karambit

nemvar commented:
Use *= 0.5

CptnCapitalism committed:
Changed "/=2" to "*=0.5"

optimumtact commented:
Floyd we have literally no way to adequately balance antag traitors anymore precisely because of that issue

Qustinnus commented:
Mmh that sounds fair but I feel like there's still ways to get around that by introducing a system that circumvents the balance problems

ExcessiveUseOfCobblestone commented:
Find someone who will merge this as-is or have it replace another uplink item, I'm not really interested in more traitor options that are just another way to directly kill a player.

ExcessiveUseOfCobblestone reviewed (changes requested):
above

anconfuzedrock commented:
I dunno why you'd use this thing to kill another person since this is just a hatchet in power but the rad health scanner is rarely used and extremely unfun for the victim and unengaging for the user.

ExcessiveUseOfCobblestone commented:
sounds like a good replacement item then.
